Trading/Mode/StaggeredOrdersTradingMode.json

```diff
@@ -1,10 +1,21 @@
 {
   "required_strategies": ["StaggeredOrdersStrategiesEvaluator"],
-  "mode": "mountain",
-  "spread_percent": 4,
-  "increment_percent": 3,
-  "lower_bound": 300,
-  "upper_bound": 8000,
-  "allow_instant_fill": true,
-  "operational_depth": 100
+  "BTC/USDT": {
```

Comment:
That's examples ? Or it's required ? So now we'll need to create a symbol specific configuration or we can continue to use generic configuration ?

Reply:
You have to make symbol specific configurations, the only change now is that you need to specify the traded pair as well (since prices were depending on the selected pair already, it's not a big change